hugo.yaml

```diff
@@ -114,8 +114,8 @@ params:
         Thank you. Your feedback is appreciated!
       # prettier-ignore
       'no': >-
-        Please let us know <a class="external-link" target="_blank" rel="noopener"
-        href="https://github.com/open-telemetry/opentelemetry.io/issues/new?title=Page%20feedback&body=Suggested%20improvements%20for%20page:%20ADD%20PAGE-URL-HERE">how
+        Please let us know <a class="external-link feedback-link" target="_blank" rel="noopener"
```

Comment:
What is the `feedback-link` class for?

Reply:
Ah, I was shortly thinking about doing some javascript hack to update the url parameter with the current url but then decided against it and waiting for docsy to provide that as a hugo variable as an upstream feature ;-)